OktaAspNetExample/Views/Shared/_Layout.cshtml

```diff
@@ -28,7 +28,7 @@
                 {
                     <ul class="nav navbar-nav navbar-right">
                         <li>
-                            <p class="navbar-text">Hello, <b>@Context.User.Identity.Name</b></p>
+                            <p class="navbar-text">Hello, <b>@ViewBag.Username</b></p>
```

Comment:
This only works on the Home-Index page, because that's the only action that sets `ViewBag.Username`.

Let's go back to `@Context.User.Identity.Name`. It wasn't showing up because of [this issue](https://stackoverflow.com/q/29973963/3191599) which I'll fix in a commit.